doc/content/getting_started/RunningBlackBear.md

```diff
@@ -32,20 +32,29 @@ cd ~/projects/
 git clone git@github.com:idaholab/blackbear.git
 ```
 
-Next initialize the MOOSE submodule:
+BlackBear requires a checkout of the MOOSE repository to link against. By default, the BlackBear
+build process looks for that in a directory called `moose` located in the same directory
+that contains the `blackbear` directory. Clone the MOOSE repository using the following command:
 
 ```bash
-cd ~/projects/blackbear/
-git submodule update --init
+cd ~/projects/
+git clone git@github.com:idaholab/moose.git
 ```
 
```

Comment:
I'd recommend including an example of how the projects directory should look, with both moose and blackbear, or perhaps a line about verifying that blackbear and moose appear in the projects directory

Reply:
Done